package.json

```diff
@@ -28,7 +28,7 @@
 		"lint": "intern-dev-lint",
 		"build": "intern-dev-clean && concurrently intern-dev-build intern-dev-api",
 		"release": "intern-dev-release",
-		"test": "intern-dev-test",
+		"test": "intern-dev-clean && intern-dev-build && intern-dev-test all config=_build/tests/intern-local",
```

Comment:
I'd prefer to leave the `test` command as 'intern-dev-test' for now since it's easier to change the config from the command line then.

Reply:
@jason0x43 done.